### DISCUSSION TITLE
Support importing seed phrase via SeedQR

### DISCUSSION THREAD

**Discussion by sethforprivacy** [ACINQ/phoenix#446] [Improving Phoenix]
It would be fantastic to be able to import a seed into Phoenix using the [SeedQR format](https://github.com/SeedSigner/seedsigner/blob/dev/docs/seed_qr/README.md).

With Passport, you are able to create child seeds using BIP 85 and export them in SeedQR format, allowing you to keep one single backup (of the master seed for Passport) and restore all of your Nostr keys and mobile wallet seeds directly from that master backup. We've written more on that functionality here: https://foundationdevices.com/2023/05/all-your-wallets-one-backup/

It would be fantastic to be able to simply scan in a seed phrase when setting up Phoenix so that the recovery phrase can be a child seed of an existing hardware wallet, removing the need for an additional backup.

Please don't hesitate to let me know if you need anything else around this from us, we're happy to help however we can!

**Comment by dpad85** [comment]
Thanks for this suggestion, it sounds great. We won't be able to work on it anytime soon, but it could be a nice advanced option to support in the future, in addition to #391 (border wallets).
